zrml/prediction-markets/src/lib.rs

```diff
@@ -1576,6 +1557,8 @@ mod pallet {
         /// Someone is trying to call `dispute` with the same outcome that is currently
         /// registered on-chain.
         CannotDisputeSameOutcome,
+        /// The market dispute mechanism has not failed.
+        DisputeMechanismHasNotFailed,
```

Comment:
A test should cover this if possible. Also as per @sea212 's latest comment on other PR, we should add new errors to end of the enum definition so that it does not create issues in metadata for FE. See https://github.com/zeitgeistpm/zeitgeist/pull/894#discussion_r1063636163

Reply:
Good catch. I will think about this next time. The error was in use before all of the reviews and now I removed it, because it was unused. 7ba0801690667e3d0383f2779296e979cd6859b0